counter/src/main/java/edu/nextstep/camp/counter/CounterActivity.kt

```diff
@@ -1,11 +1,27 @@
 package edu.nextstep.camp.counter
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import edu.nextstep.camp.counter.databinding.ActivityCounterBinding
 
 class CounterActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityCounterBinding
+    private val viewModel: CounterViewModel by viewModels()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_counter)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_counter)
+        binding.lifecycleOwner = this
+        binding.viewModel = viewModel
+
+        viewModel.eventShowToast.observe(this) { stringId ->
+            stringId?.let {
+                Toast.makeText(this, stringId, Toast.LENGTH_SHORT).show()
+            }
```

Comment:
```suggestion
            Toast.makeText(this, stringId ?: return, Toast.LENGTH_SHORT).show()
```
이런식으로도 활용해볼 수 있겠네요 🙂